elide-graphql/src/main/java/com/yahoo/elide/graphql/Entity.java

```diff
@@ -32,19 +33,24 @@
     @Getter private RequestScope requestScope;
     @Getter private Set<Attribute> attributes;
     @Getter private Set<Relationship> relationships;
+    @Getter private EntityProjection projection;
```

Comment:
This would be pointer to a constructed projection. To be used in `toPersistenceResource()` method which needs a projection and `setRelationships()` method in which we can get the `Relationship` object from the projection without constructing it again.